editor-components/EditorComponent.tsx

```diff
@@ -535,6 +535,16 @@ export abstract class BaseList extends Component {
 
 export abstract class BaseHeader extends Component {
   static category = CATEGORIES.HEADER;
+
+  insertForm(name: string, data: Object) {
+    const project = getProjectHook()._id;
+    let config = {
+      ...{ data: { name, data, project } },
+      method: "post",
+      url: process.env.REACT_APP_API_URL + "/fn-execute/project/insert-form",
+    };
+    return axios.request(config).then((r: any) => r.data);
+  }
```

Comment:
There is no need for this function. Cause we are already added to insert form function for all component classes. remove it